tests/audiohelper.js

```diff
@@ -100,6 +100,10 @@ describe('AudioHelper', () => {
         mediaDevices,
         setSinkId: () => {}
       });
+      
+      navigator.mediaDevices = {
+        enumerateDevices() { return Promise.resolve([]) },
+      };
```

Comment:
Since we're setting this in beforeEach, should we save the original, then restore it in afterEach?